Felienne/hedy
# [FEATURE] Allows teachers to duplicate classes

**Description**
In this PR we introduce a new feature, enabling teachers to duplicate classes. This can be useful when having an already customized classes and wanting to copy this for another class. When duplicating all customizations are automatically copied to the new class.

**Fixes**
This PR fixes #3115.

**How to test**
Make sure you are a teacher and navigate to the `/for-teachers` page. Notice the new duplicate option and verify that it works and copies all customizations.
